frontend/src/composables/WorkSpace/useDomainData.js

```diff
@@ -141,28 +152,40 @@ export default function useDomainData() {
 	getTagsTree();
 
 	const OnDelete = async (id) => {
-		await API.delete(`/domain/${choosenNodeID.value}/tag/${id}`)
-			.then(() => refreshDomainTag(choosenNodeID.value))
+		try {
+			await API.delete(`/domain/${choosenNodeID.value}/tag/${id}`);
+			await refreshDomainTag(choosenNodeID.value);
+			quasar.notify({
+				type: "positive",
+				message: "Tag has been successfully deleted",
+			});
+		} catch (error) {
+			quasar.notify({
+				type: "negative",
+				message: "Sorry, tag has not been deleted",
```

Comment:
Same comment as above

Reply:
The Error message should be humain readable. but Error whitle adding tag works as well :)